### ISSUE TITLE
8-digit code is not displayed on the screen as users type (Android)

### ISSUE THREAD

**Issue by sarasuri** [cds-snc/covid-shield-mobile#274]
Scenario to reproduce
Samsung Galaxy J3 Prime
Android version 7.0
Display orientation: Landscape

When users try to enter an 8-digit code on the “Enter your 8-digit number” screen in landscape orientation the number is not displayed on the screen as the users type. They can only see what they entered after pressing on the “Done” button.

Expected result
Users should be able to see the numbers they type in the field.

"WCAG
https://www.w3.org/WAI/WCAG21/Understanding/orientation

Screenshots
![digitp](https://user-images.githubusercontent.com/12256067/85968336-da79ca80-b992-11ea-9596-8f2778e7524c.gif)
![digitl](https://user-images.githubusercontent.com/12256067/85968342-dd74bb00-b992-11ea-9264-7005c9fc66e9.gif)

**Comment by JuliannaR** [comment]
Considerations for known issues around orientation. 
Connected upstream to https://github.com/CovidShield/mobile/issues/11

**Comment by smcmurtry** [comment]
We locked to portrait mode in #297
